### ISSUE TITLE
v9 Presence Badge: colors are missing in Windows contrast mode

### ISSUE THREAD

**Issue by smhigley** [microsoft/fluentui#23152]
We should preserve the presence badge colors in WHCM -- they're very small icons, and the color is meaningful and pretty helpful. I think this would be much easier to visually parse with colors:

![screenshot of a row of avatars with different presence badges. All badges have a tiny white icon over dark blue background](https://user-images.githubusercontent.com/3819570/169897909-87c001e7-f55e-44ab-8734-c75dc8796331.png)

While generally we don't want to override text/bg colors, in this case since the badges are functioning more like images, my judgement would be that they would be much easier to understand with the original colors.

**Comment by jurokapsiar** [comment]
Alternative would be to provide bigger icons in HC theme. I think it will be hard to maintain contrast ratio, especially for the new and custom W11 contrast themes.

**Comment by smhigley** [comment]
Re-opening this, I think I'll try updating the icons so that the borders keep the original color, while the bg & icon adopt HCM colors
